package.json

```diff
@@ -8,7 +8,7 @@
     "test:lint": "eslint ./ --ignore-path .gitignore --ext .js",
     "test:lockfile": "lockfile-lint -p package-lock.json -t npm -a npm -o https: -c -i",
     "test:git-history": "commitlint --from origin/main --to HEAD",
-    "test:unit": "jest",
+    "test:unit": "NODE_ENV=production jest",
```

Comment:
do you know which was the brittle test ? ideally any test that relies on NODE_ENV being set to production should have it set so within the test

Reply:
Likely the last PR that went in, not sure why the build passed on the PR though

Reply:
yeah the last PR that went in, it didn't fail because we set the node_env properties on github workflows for tests but that is not the case for the release workflows